src/lib/pages/backoffice/Loan/LoanDetails/LoanActionMenu/LoanActionMenu.js

```diff
@@ -12,7 +12,7 @@ export default class LoanActionMenu extends Component {
   render() {
     const { loan, offset } = this.props;
     return (
-      <div className="bo-action-menu">
+      <div>
```

Comment:
why is the class removed?

Reply:
It was adding an extra margin to the action buttons and it was ruining the display of the overdue reminder button. I removed it but the margins are still fine:

<img width="308" alt="Screenshot 2021-03-30 at 15 32 36" src="https://user-images.githubusercontent.com/33685068/112997196-2c8f8c80-916d-11eb-9773-597dd8228230.png">